science/gnss-sdr/Portfile

```diff
@@ -46,33 +51,35 @@ subport gnss-sdr-devel {
     checksums           rmd160 733c3211b163825be36db80d89f9fb0db0051264 \
                         sha256 d44b32fd2bbdc703097e2368281d77ad4e2c42ec7c76c6e7ef100b014a716e3e \
                         size   3575803
-    revision            1
+    revision            2
 
     conflicts           gnss-sdr gnss-sdr-next
 
     depends_lib-append  port:gnuradio-devel port:volk-gnss-sdr-devel
 
     require_active_variants port:gnuradio-devel uhd
 
+    patchfiles          patch-boost-asio.diff
+
 }
 
 subport gnss-sdr-next {
     long_description    ${description}: \
-        This port is kept up with the GNSS-SDR GIT next branch, which is typically updated daily to weekly.  This version of GNSS-SDR generally contains fixes and new features that will be incorporated in an upcoming release, and compiles against the gnuradio-next port, which represents GNU Radio GIT next branch.  This port may or not compile or function correctly, as it represents a work in progress.  If it does not work, check back in a few days.  Or try deactivating the currently active gnss-sdr and gnuradio ports, cleaning any current builds, and trying again.
+        This port is kept up with the GNSS-SDR GIT next branch, which is typically updated daily to weekly.  This version of GNSS-SDR generally contains fixes and new features that will be incorporated in an upcoming release, and compiles against the gnuradio-devel port, which represents GNU Radio development branch.  This port may or not compile or function correctly, as it represents a work in progress.  If it does not work, check back in a few days.  Or try deactivating the currently active gnss-sdr and gnuradio ports, cleaning any current builds, and trying again.
 
     name                gnss-sdr-next
-    github.setup        gnss-sdr gnss-sdr 40b7377c318ceeb72983069e23a1ba4e8908b526
-    version             20190516-[string range ${github.version} 0 7]
-    checksums           rmd160 88d5e31136871a1c865f6e62377dd216669f9876 \
-                        sha256 f775db42efd2cf75f028a11de50fea05009ad7a7ed803077d7971d1bd9fbd7fb \
-                        size   3689595
+    github.setup        gnss-sdr gnss-sdr d13e00ba502462e4b498b7fcc0dc9f965df48edf
+    version             20190523-[string range ${github.version} 0 7]
+    checksums           rmd160 ee32e0d2c9666644eab648f3c37c16afc5faa8ba \
+                        sha256 4bae6b37968a726a3f9a67ca0beae7691a0f20885111559d5cd6d4dca34394ff \
+                        size   3689496
     revision            0
 
     conflicts           gnss-sdr gnss-sdr-devel
 
-    depends_lib-append  port:gnuradio-next port:volk-gnss-sdr-next port:lapack
+    depends_lib-append  port:gnuradio-devel port:volk-gnss-sdr-next port:lapack port:protobuf3-cpp
```

Comment:
Please add a comment as to why gr-devel, not next; your hope for using next in the future once <something?> happens ... I thought the "next" here was supposed to follow the "next" there ... so not sure why this change?

Reply:
Here the intent is to build the latest version of GNSS-SDR (which is in the next branch) against the latest GNU Radio development, which used to be gnuradio-next but now, from my understanding, it is gnuradio-devel. 

This also saves gr-osmosdr, which cannot be built against gnuradio-next.

I guess that, once GNU Radio 3.8 is out, new GNU Radio developments will return back to "gnuradio-next". Until then, the combination gnss-sdr-next with gnuradio-devel seems more usable.

I will add comments explaining this, if you find it ok.

Reply:
Actually just "gnuradio" is the current latest 3.7 code; "gnuradio-devel" is behind a little currently; I need to get that dealt with one of these days.

"gnuradio-next" is the latest 3.8 code.

So ... if you want to support 3.8, then "next" is the way to go. If you want to support 3.7, choose either of the other gnuradio ports.

Reply:
ok, then I'll switch back to gnuradio-next.